run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
NVM this only works for unix but we have to do this platform independent :smirk: